apps/web-giddh/src/app/contact/contact.component.html

```diff
@@ -783,7 +775,7 @@ <h1>{{commonLocaleData?.app_no_record_found}}</h1>
                                         <label></label>
                                         <input type="checkbox" (click)="selectAccount($event, account) "
                                             [(ngModel)]="account.isSelected "
-                                            [ngClass]="{ 'checkbox-visible': (selectedCheckedContacts.length || isMobileScreen), 'position-absolute': !selectedCheckedContacts.length} " />
```

Comment:
Class name getting replaced. Please check

Reply:
They do the same thing `position-absolute` is of Bootstrap and `pos-abs` is our custom. Will create a new PR that will replace all the class names with Bootstrap class in all the env.